### PR TITLE
Update streaming subscription to understand new initial response

### PR DESCRIPTION
**Pull request by JoshVanL**
# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/validate_examples.yaml [hunk 1]
```diff
@@ -150,7 +150,7 @@ jobs:
       GOPROXY: https://proxy.golang.org
       DAPR_INSTALL_URL: ${{ needs.setup.outputs.DAPR_INSTALL_URL }}
       DAPR_CLI_VER: ${{ needs.setup.outputs.DAPR_CLI_VER }}
-      DAPR_RUNTIME_VER: 1.14.0-rc.2
+      DAPR_RUNTIME_VER: 1.14.0-rc.4
```
Comment (mikeee) [inline .github/workflows/validate_examples.yaml:153]:
```suggestion
      DAPR_RUNTIME_VER: ${{ needs.setup.outputs.DAPR_RUNTIME_VER }}
```

File: .github/workflows/validate_examples.yaml [hunk 2]
```diff
@@ -32,15 +32,15 @@ jobs:
...
       GOPROXY: https://proxy.golang.org
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/master/install/install.sh
-      DAPR_CLI_REF: 19b9de05611ade540b06d2c061f32f6c37093a17
+      DAPR_CLI_REF: e72f95393cc0c66b0cc4d726f5c45f80b916b400
       DAPR_REF: ${{ github.event.inputs.daprdapr_commit }}
       CHECKOUT_REPO: ${{ github.repository }}
       CHECKOUT_REF: ${{ github.ref }}
     outputs:
       DAPR_INSTALL_URL: ${{ env.DAPR_INSTALL_URL }}
       DAPR_CLI_VER: ${{ steps.outputs.outputs.DAPR_CLI_VER }}
```
Comment (mikeee) [inline .github/workflows/validate_examples.yaml:41]:
```suggestion
      DAPR_CLI_VER: 1.14.0-rc.6
```

File: .github/workflows/validate_examples.yaml [hunk 3]
```diff
@@ -32,15 +32,15 @@ jobs:
       GOARCH: amd64
       GOPROXY: https://proxy.golang.org
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/master/install/install.sh
-      DAPR_CLI_REF: 19b9de05611ade540b06d2c061f32f6c37093a17
+      DAPR_CLI_REF: e72f95393cc0c66b0cc4d726f5c45f80b916b400
```
Comment (mikeee) [inline .github/workflows/validate_examples.yaml:35]:
```suggestion
      DAPR_CLI_REF: ${{ github.event.inputs.daprcli_commit }}
```

**Review by mikeee — COMMENTED** [review]
Just some suggestions for the workflow

**Comment by JoshVanL** [comment]
@mikeee done

**Review by mikeee — APPROVED** [review]

**Pull request merged**